### ISSUE TITLE
New Feature Suggestion - Respawn Claim

### ISSUE THREAD

**Issue by fclaussen** [Galarzaa90/NabBot#33]
TS3 bot have a respawn claim functionality.
This is something major that would really bring this project to the next level

I'll try to explain how it works and provide as much information as possible.

You send a pm to the TS3 bot with `!resp xxx` where `xxx` is a reference number to a respawn. See attached image below
![image](https://user-images.githubusercontent.com/1021860/27814199-579c5b52-6051-11e7-8362-7ff4be63e9c7.png)

Then, on a different channel you can see who is on which respawn. like so:
![image](https://user-images.githubusercontent.com/1021860/27814208-674a0496-6051-11e7-9f48-6f75624fa354.png)

I imagine this just like the /watched. A bot message on a channel that is edited constantly.

This is reset at save server.
If I am not on the respawn anymore, you run `!respdel xxx`

## List of commands
!resp X - Add yourself to the respawn
!respdel X - Remove yourself from the respawn 
!respnext X - Add yourself as the next on the respawn
!respnextdel X - Remove yourself as the next on the respawn

**Comment by Galarzaa90** [comment]
Yes, this is something that has been requested a couple times and I want to add, but I'm I haven't decided on the details of how to implement it and I been focusing on other little details (when I have time).

**Comment by Galarzaa90** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Respawn claim](https://trello.com/c/LyyBXjpS/4-respawn-claim)
